### DISCUSSION TITLE
[wayland] support running as a nested compositor

### DISCUSSION THREAD

**Discussion by smurfix** [qtile/qtile#4463] [Ideas]
Starting `sway` while Wayland is running opens a nice nested compositor window, which is ideal for testing and whatnot (assuming you remember to use a different binding for the Mod key …).

Starting `qtile --backend=wayland` the same way briefly flickers a window, but then qtile just terminates silently.

I'd be ecstatic if this would work. ;-)

NB, the `--backend=wayland` should not be necessary either. If it's omitted qtile tries to run as an X11 window manager, which fails for obvious reasons.

**Comment by micimize** [comment]
I think the docs are misleading here, as they [imply support](https://docs.qtile.org/en/stable/manual/install/index.html#wayland) for this that in my brief testing doesn't seem to work:
> With the Wayland dependencies in place, Qtile can be run either from a TTY, or within an existing X11 or Wayland session where it will run inside a nested window:
> `qtile start -b wayland`
